programmers-coding-test/minjae/Implement/구현-왕실의-게임.js

```diff
@@ -0,0 +1,25 @@
+function solution(n, m) {
+    const chessLength = 8;
+    const x = n.charCodeAt(0) - 'a'.charCodeAt(0) + 1;
+    const y = m.charCodeAt(0) - '0'.charCodeAt(0);
+    const stepList = [
```

Comment:
List, Array, Map, Set, Split 등과 같은 이름은 변수명에 넣어주는걸 최대한 피해주세요!
```suggestion
    const steps = [
```